### PR TITLE
Import local CA into p12 file to fix zosmf jwt SSO issue - target to rc 

### PR DESCRIPTION
**Pull request by vit-tomica**
Signed-off-by: Vit Tomica <vit.tomica@broadcom.com>

The zosmf JWT is inserted into fake certificate that is signed by Zowe local CA. The zowe local CA has to imported to p12 file as well before the p12 is imported to pkcs11 token using gskkyman.

Related PR: https://github.com/zowe/api-layer/pull/679

### CODE REVIEW

**Comment by plavjanik** [comment]
Requires https://github.com/zowe/api-layer/pull/679

File: bin/apiml_cm.sh [hunk 1]
```diff
@@ -330,7 +330,8 @@ function zosmf_jwt_public_key {
         -cp ${BASE_DIR}/../components/api-mediation/gateway-service.jar \
         -Dloader.main=org.zowe.apiml.gateway.security.login.zosmf.SaveZosmfPublicKeyConsoleApplication \
         org.springframework.boot.loader.PropertiesLauncher \
-        https://${ZOWE_ZOSMF_HOST}:${ZOWE_ZOSMF_PORT} ${SERVICE_KEYSTORE}.${JWT_ALIAS}.pem
+        https://${ZOWE_ZOSMF_HOST}:${ZOWE_ZOSMF_PORT} ${SERVICE_KEYSTORE}.${JWT_ALIAS}.pem "${LOCAL_CA_ALIAS}" \
+        ${LOCAL_CA_FILENAME}.keystore.p12 PKCS12 ${LOCAL_CA_PASSWORD} ${LOCAL_CA_PASSWORD}
```
Comment (vit-tomica) [inline bin/apiml_cm.sh:334]: Two occurrences of ${LOCAL_CA_PASSWORD}  ... is it intentional?

Comment (plavjanik) [reply bin/apiml_cm.sh:334]: yes, first one is keyStorePassword, the second one is keyPassword

File: bin/zowe-setup-certificates.sh [hunk 2]
```diff
@@ -156,6 +156,7 @@ if ! [[ -z "${PKCS11_TOKEN_NAME}" ]] && ! [[ -z "${PKCS11_TOKEN_LABEL}" ]]; then
     if ! keytool -importcert -file ${APIML_PUBLIC_KEY} -keystore ${P12_PUBLIC_KEY} -storetype pkcs12 -storepass ${KEYSTORE_PASSWORD} -trustcacerts -noprompt >> $LOG_FILE 2>&1 ; then
       echo "Unable to convert ${APIML_PUBLIC_KEY} to PKCS#12. See $LOG_FILE for more details."
     else
+      keytool -importcert -file ${LOCAL_CA_PREFIX}.cer -alias localca -keystore ${P12_PUBLIC_KEY} -storetype pkcs12 -storepass ${KEYSTORE_PASSWORD} -trustcacerts -noprompt >> $LOG_FILE 2>&1
```
Comment (plavjanik) [inline bin/zowe-setup-certificates.sh:159]: Will this work well when the certificate is not signed by localca?

Comment (vit-tomica) [reply bin/zowe-setup-certificates.sh:159]: I've just tested that in that case the localca cert is ignored and not imported.

**Pull request merged**